rs/nns/governance/canister/governance.did

```diff
@@ -809,6 +921,18 @@ type ProposalData = record {
   wait_for_quiet_state : opt WaitForQuietState;
   executed_timestamp_seconds : nat64;
   original_total_community_fund_maturity_e8s_equivalent : opt nat64;
+
+  // This is the baseline that determines the amount of voting rewards (in the
+  // form of additional maturity) that voting neurons receive.
+  //
+  // For example, suppose neuron N has 42 voting power in ballot B for this
+  // proposl. The amount of voting power in B is determined by the neuron's
+  // deciding voting power when the proposal was created. If this field has 99,
+  // then, neuron N receives 42 / 99 of the rewards purse for this proposal.
+  //
+  // Note that this is NOT used to determine whether the proposal is adopted or
```

Comment:
How about we state it positively 
"Note that the number used to determine whether a proposal is adopted is ...."  ?

Reply:
What is this other number, the actual voting power? Is it not appearing in any of the APIs as it is just implicitly defined by the ballots in a proposal?

Reply:
> How about we state it positively

👍 Let's do both.

I want to emphasize what this does NOT do, because it is very easy to incorrectly think that this is used to decide a proposal.

> Is it not appearing in any of the APIs as it is just implicitly defined by the ballots in a proposal?

It's in the latest_tally field. (Yes, it can also be reconstructed from ballots.)

Reply:
Done.